sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -659,10 +665,31 @@ public void promptForRefAppVersionIfMissing(Server server, VersionsHelper versio
 		}
 	}
 
+	@Override
+	public void promptForO3RefAppVersionIfMissing(Server server, VersionsHelper versionsHelper) throws MojoExecutionException {
+		promptForO3RefAppVersionIfMissing(server, versionsHelper, null);
+	}
+
+	@Override
+	public void promptForO3RefAppVersionIfMissing(Server server, VersionsHelper versionsHelper, String customMessage)
+			throws MojoExecutionException {
+		if (server.getVersion() == null) {
```

Comment:
Where are you using the `customMessage` parameter?

Reply:
I based that method on this method : https://github.com/openmrs/openmrs-sdk/blob/545ea9cff6bfaabf8bc7645a5994945155dc3c5a/sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java#L651-L660. I don't think it is necessary, I'll remove it.